build/NetStandardTest.targets

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <NoWarn>1701;1702;CA1707</NoWarn>
+    <NoWarn>1701;1702;CA1707;RS0100</NoWarn>
```

Comment:
I thought we didn't run the analyzers on test projects?

Reply:
Removed